hc/settings.py

```diff
@@ -96,6 +96,8 @@
             'ENGINE':   'django.db.backends.postgresql',
             'NAME':     'hc',
             'USER':     'postgres',
+            'PASSWORD': 'engneerdon',
```

Comment:
no passwords in the repo

Reply:
Jesus Christ. HAHAHA. Let me remove that with immediate effect.